src/themes/jetstream/color-darkmode/alert.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
It appears that the Tier2 tokens here were based on the old version of the AuroDesignTokens. While jetstream may have unique brand and base color tokens, the rest of the tokens "must" match the tokens that are currently released in the repo. I can tell that they currently do not because --ds-color-alert... tokens were all deprecated in a recent release.

Please let me know if you need further clarification on what this means for the requested change.

Reply:
I see, thanks for letting me know, I will ask some help from the Jetstream's GD to adjust this.
Will come back to talk to you later.

Reply:
I think I wasn't clear here @fajar-apri-alaska *all* of these files with Tier 2 tokens (everything except brand.json and base.json) appear to be outdated, not just the alert file.

Reply:
Actually, I got your point on the first time, but I couldn't reach Andri (yet) as it seems He's on leave.
So I tried to double check the json again myself, with the document that He give. In the end, maybe He is the one can confirm if the value is still okay or some value needs to be updated. I will try to reach Him again today.